tt_eager/tt_dnn/op_library/transformer_tms/kernels/compute/ssm_eltwise_mul.cpp

```diff
@@ -109,6 +118,8 @@ void MAIN {
                 tile_regs_wait();
 
                 mul_bcast_rows_init_short(cb_in0_transposed, cb_in1_bcast_row);
+                unpack_reconfig_data_format(cb_in0_transposed, cb_in1_bcast_row);
```

Comment:
`unpack_reconfig_data_format(cb_in0_transposed, cb_in1_bcast_row); ` This call reprograms both srcA & srcB, but srcB format seems unchanged in this kernel, you can use:

```
/**
 * Helper function to reconfigure unpacker srca input data format.
 */
ALWI void unpack_reconfig_data_format_srca(const uint32_t srca_new_operand) {
    UNPACK(( llk_unpack_reconfig_data_format_srca(srca_new_operand) ));
}
```

Reply:
That works, thanks!